core/app/views/refinery/_header.html.erb

```diff
@@ -1,4 +1,11 @@
-<h1 id='logo'>
-  <%= link_to Refinery::Core.site_name, refinery.root_path %>
-</h1>
-<%= Refinery::Pages::MenuPresenter.new(refinery_menu_pages, self).to_html %>
+<%= content_tag :div, id: "logo", itemscope: "",  itemtype: "http://schema.org/Organization" do %>
+  <%= link_to refinery.root_path, itemprop: "url" do %>
+    <%= content_tag :span, { itemprop: "name"} do %>
```

Comment:
Finally, after reading doc (http://schema.org/name) and examples (https://github.com/Shopify/Timber/blob/master/layout/theme.liquid#L169) i prefer to not add span surrounding the site name